cocotb/drivers/amba.py

```diff
@@ -365,3 +486,15 @@ def _read_data(self):
                 self.bus.RLAST <= 0
                 if burst_count == 0:
                     break
+
+class AXI4MemoryMap(AXI4Slave):
+    def __init__(self, size, *args, **kwargs):
+        AXI4Slave.__init__(self, *args, **kwargs)
+        self._memory = memoryview(bytearray(size))
+
+    @cocotb.coroutine
+    def do_slave_write(self, addr, data_bytes):
+        self._memory[addr:addr + len(data_bytes)] = data_bytes
+
+        yield NullTrigger()
+        return 0
```

Comment:
This is a syntax error in python 2. It doesn't look like you need the 0 anyway, and `return` would be fine. If you do need the return, use `raise ReturnValue(0)` for compatibility with python 2.

Reply:
yea, the return value isn't used anymore.